modules/ocf_broker/manifests/init.pp

```diff
@@ -0,0 +1,51 @@
+class ocf_broker {
+    include ocf_ssl::default_bundle
+
+    package { ['redis-server', 'hitch']: }
+
+    service { 'redis-server':
+      require => Package['redis-server'],
+    }
+
+    file { '/etc/redis/redis.conf':
+      owner   => redis,
+      group   => root,
+      mode    => '0600',
+      require => Package['redis-server'],
+      notify  => Service['redis-server'];
+    }
+
+    $redis_password = assert_type(Pattern[/^[a-zA-Z0-9]*$/], hiera('broker::redis::password'))
+
+    augeas { '/etc/redis/redis.conf':
+        lens      => 'Spacevars.simple_lns',
+        incl      => '/etc/redis/redis.conf',
+        changes   => [
+          'set port 6379',
+          "set requirepass ${redis_password}",
+          'set appendonly yes',
+          'rm save',
```

Comment:
inconsistent quotation

Reply:
I believe to interpolate variables you need double quotes, so I think this is fine.

Reply:
Yeah, just checked, this is necessary to get this to work. For example, [this commit](https://github.com/ocf/puppet/commit/7491c0c7cfa7a03c639facd03a42c5a9db4a8406).

Reply:
I know, so we may as well use double-quotes everywhere I think, because the one line with doubles and everything else with singles is slightly triggering me lol

Reply:
Eh, I'd just leave it as is, I think having single quotes except when needing double quotes makes more sense.